README.md

```diff
@@ -42,7 +42,7 @@ Default: `'JST'`
 The namespace in which the precompiled templates will be assigned.  *Use dot notation (e.g. App.Templates) for nested namespaces or false for no namespace wrapping.*  When false with `amd` or `commonjs` option set `true`, templates will be returned directly from the AMD/CommonJS wrapper.
 
 Example:
-```js
+```javascript
```

Comment:
also let's remove the trailing spaces, they get in the way and the formatting is not suffering too much